ports/vcpkg-cmake-validate/vcpkg_cmake_validate.cmake

```
@@ -0,0 +1,161 @@
+#[===[.md:
+# vcpkg_cmake_validate
+
+Test the correctness of the cmake package information established by a find_package() call.
+
+```cmake
+vcpkg_cmake_validate(
+    [CMAKE_VERSION <version>]
+    [CMAKE_PROLOG <cmake statements>...]
```

```suggestion
    [CMAKE_PROLOGUE <cmake statements>...]
```

you should also `s/PROLOG/PROLOGUE` generally

---

FTR there used to be `PROLOG` in https://cmake.org/cmake/help/v3.21/module/WriteCompilerDetectionHeader.html?highlight=prolog.

---

Done.